code/game/machinery/kitchen/cooking_machines/container.dm

```diff
@@ -48,13 +48,23 @@
 	set category = "Object"
 	set desc = "Removes items from the container. does not remove reagents."
 
+	if (!isliving(usr))
+		usr << "Ghosts can't mess with cooking containers"
```

Comment:
Do we really need to give a feedback message to ghosts? Most things don't have one.